src/workspaces/index.ts

```diff
@@ -123,6 +124,12 @@ export class Workspaces {
     );
   }
 
+  public getWorkspaceUsage(
+    workspaceId: string,
+  ): Promise<{ totalWorkspaceSpace: number; spaceAssigned: number; spaceUsed: number }> {
```

Comment:
Perhaps better to add the type of the data to return for use in the project using the sdk

Reply:
Indeed. Done!